### PR TITLE
gelu tensorflow frontend and its test

### PR DESCRIPTION
**Pull request by Mina-88**
#5521

### CODE REVIEW

**Comment by ahmedo42** [comment]
Hi @Mina-88 , Thanks for your contribution!
can you fix the lining errors and ensure that you discard any changes unrelated to your function?

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_activations.py [hunk 1]
```diff
@@ -106,3 +107,33 @@ def test_tensorflow_softmax(
         x=x[0],
         axis=axis,
     )
+
+
+# gelu test
+@handle_cmd_line_args
+@given(
+    dtype_x_and_approximate=helpers.dtype_and_values(
```
Comment (ahmedo42) [inline ivy_tests/test_ivy/test_frontends/test_tensorflow/test_activations.py:115]: this should be `dtype_and_x`

Comment (Mina-88) [reply ivy_tests/test_ivy/test_frontends/test_tensorflow/test_activations.py]: Thank you so much!

**Comment by ahmedo42** [comment]
LGTM @Mina-88! since the failures are from our side we can merge.

**Comment by Mina-88** [comment]
Thank you!

**Pull request merged**